### ISSUE TITLE
Is there any validation split?

### ISSUE THREAD

**Issue by smiles724** [molML/MoleculeACE#5]
Hi, thanks for sharing the code. 

However, as far as I am concerned, you only split the data into training and testing, but ignore the validation split. 

It is important to have both train and validation, otherwise, you will not have enough knowledge to know when to stop training and what is the best model. I believe cross-validation cannot avoid the drawback of missing the validation split.

Can you please give me a potential answer?

**Comment by githubXin123** [comment]
@smiles724 Hello, the author employed early stopping strategy in cross-validation to mitigate overfitting.

**Comment by smiles724** [comment]
> @smiles724 Hello, the author employed early stopping strategy in cross-validation to mitigate overfitting.

I am afraid an early stop is not the correct way to select the best model. In other words, you should evaluate the training process based on the validation set, which is missing.

**Comment by githubXin123** [comment]
@smiles724 First, the author divided the dataset into a training set and a test set. Then, the author applied 5-fold cross-validation to the training set, where in each fold, one-fifth of the data was used as a validation set to evaluate the model.

**Comment by nibingwei123** [comment]
> [@smiles724](https://github.com/smiles724) First, the author divided the dataset into a training set and a test set. Then, the author applied 5-fold cross-validation to the training set, where in each fold, one-fifth of the data was used as a validation set to evaluate the model.

Hello, I would like to know how the test RMSE in the MoleculeACE_results.csv file was obtained. Was it obtained by training for x epochs, where x is the number of epochs determined by five-fold cross-validation, without a validation set?
